tests/integration/targets/mongodb_parameter/tasks/main.yml

```diff
@@ -43,10 +43,12 @@
 - name: Kill all mongod processes
   command: pkill  -{{ kill_signal }} mongod
   ignore_errors: true
+
 - name: Getting pids for mongod
   register: pids_of_mongod
-  community.general.pids:
+  community.general.pids::
```

Comment:
```suggestion
  community.general.pids:
```

Reply:
Doh! That's me burning the midnight oil too much! Thanks again!

Reply:
I have random `:wq` or `i` in files from using Vim, we've all been there.